services/blockchain-connector/tests/functional/decoder.test.js

```diff
@@ -76,14 +83,22 @@ describe('Functional tests for decoder', () => {
 		expect(result).toMatchObject(block);
 	});
 
-	it('decode event payload', async () => {
-		const result = await broker.call('connector.decodeEventPayload', {
+	it('decode subscription event payload', async () => {
+		const result = await broker.call('connector.decodeSubscriptionEventPayload', {
 			eventName: 'app_newBlock',
 			payload: { block: encodedBlock },
 		});
 		expect(result).toMatchObject(block);
 	});
 
+	it('decode event payload for token:transferEvent', async () => {
```

Comment:
Please rename the method as suggested and implement it as a unit test. `decodeEvent` is a pure function.

Reply:
Same as above.